source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -12,7 +13,28 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - >= 4.0
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+
+   * - >= 3.6
```

Comment:
I can't comment on this here, but I noticed when looking at the staged page that the "Language Compatibility" doesn't list 3.6. Additionally, that grid should include Node 14.x support, which has compat for the whole 3.x line

Reply:
I staged it late last night and didn't update the link until now. I'll add the Node 14.x support and update the staging links (based on the commit hash) again.

Reply:
Hm still looks like there is a missing check mark for 4.4 compat with 3.6+, also there is no `3.6.x` in the language support table